backend/document-service/src/main/kotlin/com/nosql/document/entity/ConfidentialEntity.kt

```diff
@@ -0,0 +1,22 @@
+package com.nosql.document.entity
+
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.core.mapping.Field
+
+class ConfidentialEntity(
```

Comment:
I think you should not use this `entity` in `document-service`

Reply:
fixed